code/game/objects/structures/stool_bed_chair_nest/chairs.dm

```diff
@@ -147,27 +147,27 @@
 	step(src, direction)
 	if(buckled_mob)
 		buckled_mob.dir = dir
-		switch(buckled_mob.dir)
-			if(NORTH)
-				buckled_mob.dir = SOUTH
-			if(WEST)
-				buckled_mob.dir = EAST
-			if(SOUTH)
-				buckled_mob.dir = NORTH
-			if(EAST)
-				buckled_mob.dir = WEST
+		if(!istype(src, /obj/structure/bed/chair/withwheels/wheelchair))
```

Comment:
@Carbonhell you might let me know if this is too "snowflake". I'm not sure if there's any other objects like the wheelchair.

I was just thinking we could have a flag/variable on the object saying whether if should face forward/backward when moving, which would allow new objects to use this feature without having to change this line specifically. What are your thoughts?